File: src/test/fixtures/cpp/cyclomatic_branches.cpp
// clean — not flagged by cyclomatic complexity
int classify(std::string value) {
    switch (value.size()) {
        case 1:
            return 1;
        case 2:
            return 2;
        default:
            return 0;
    }
}

// clean — not flagged by cyclomatic complexity
int classifyWithoutFallback(std::string value) {
    switch (value.size()) {
        case 1:
            return 1;
        case 2:
            return 2;
    }
    return 0;
}

File: src/test/fixtures/kotlin/CyclomaticBranches.kt
// clean — not flagged by cyclomatic complexity
fun classify(value: String): Int =
    when (value) {
        "a" -> 1
        "b" -> 2
        else -> 0
    }

// clean — not flagged by cyclomatic complexity
fun classifyWithoutFallback(value: String): Int {
    return when (value) {
        "a" -> 1
        "b" -> 2
    }
}
